Modules/Package.swift

```diff
@@ -6,6 +6,7 @@ let package = Package(
     name: "Modules",
     platforms: [
         .iOS(.v16),
+        .macOS(.v14), // Here just to avoid errors when building directly with `swift build`
```

Comment:
Is this addition necessary? Most of our packages do not support macOS and won't compile with it as a target.

Reply:
I find it handy to have just so `swift build` throws legitimate errors with the code rather than with the versions. 

In particular, it was necessary to run the WordPressFlux tests via `swift test`. However, as I try to reproduce, I can't get past some UIKit errors, which I find odd when focused on WordPressFlux, but don't have bandwidth to investigate.

Removed it.

Reply:
Thanks! Yeah, I'd remove it since it doesn't support macOS.